aptos-move/block-executor/src/proptest_types/types.rs

```diff
@@ -467,6 +549,32 @@ impl<V: Into<Vec<u8>> + Arbitrary + Clone + Debug + Eq + Sync + Send> Transactio
         )
     }
 
+    pub(crate) fn materialize_groups<
```

Comment:
This function doesn't seem to be doing anything. Where is materialization happening here?

Reply:
it wasn't implemented yet, this method can be used to take a generated mock txn without group reads/writes, and convert it to have group operations (by converting some reads / writes, so it still is derived from proptest state). Added a comment as well.